src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userservice = new UserService();
+        User check = userservice.findByEmail(email);
+        if (check == null) {
+            return false;
+        }
+        if (email.equals(check.getEmail()) && password.equals(check.getPassword())) {
+            return true;
+        }
```

Comment:
we can simplify it to this
```suggestion
        return user != null && user.getPassword().equals(password)
```
since email is basically already checked, and we can invert the null check to fit it inside one single check and return it right away without any `if`s :)